test/test_manifest.json

```diff
@@ -367,7 +367,7 @@
     },
     {  "id": "ecma262-pdf",
        "file": "pdfs/ecma262.pdf",
-       "md5": "763ead98f535578842891e5574e0af0f",
+       "md5": "8f79fca8c8a632abc6c68114bc73154e",
```

Comment:
I'd suggest changing the link to point an older of the file instead, since that way we don't have to spend time verifying that the new version of the file still exhibits the same issue: http://web.archive.org/web/20110902042030/http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf.

Not to mention that the latest version of the file has twice the number pages, and that the file size is almost four times larger.